bundles/org.openhab.binding.roku/README.md

```diff
@@ -83,7 +87,7 @@ InputHDMI3
 InputHDMI4  
 InputAV1  
 PowerOff  
-PowerOn  
+POWERON *NOTE: POWERON needs to be completely capitalized due to a bug with older Roku devices*
```

Comment:
```suggestion
POWERON _(NOTE: POWERON needs to be completely capitalized due to a bug with older Roku devices)_
```

If not parenthesis, consider some other separator as it looks a bit compacted:

![image](https://github.com/openhab/openhab-addons/assets/19519842/0b2070e7-51ce-4ff4-949f-5a9d2f335c33)